llvm/lib/CodeGen/PreISelIntrinsicLowering.cpp

```diff
@@ -560,13 +560,15 @@ bool PreISelIntrinsicLowering::lowerIntrinsics(Module &M) const {
       break;
     case Intrinsic::exp:
     case Intrinsic::exp2:
+    case Intrinsic::log:
       Changed |= forEachCall(F, [&](CallInst *CI) {
         Type *Ty = CI->getArgOperand(0)->getType();
         if (!isa<ScalableVectorType>(Ty))
           return false;
         const TargetLowering *TL = TM->getSubtargetImpl(F)->getTargetLowering();
         unsigned Op = TL->IntrinsicIDToISD(F.getIntrinsicID());
-        if (!TL->isOperationExpand(Op, EVT::getEVT(Ty)))
+        if (Op == ISD::DELETED_NODE ||
```

Comment:
This can be an assert, the pass is only explicitly handling a list of intrinsics that should always be handled in IntrinsicIDToISD